crates/holo_hash/src/fixt.rs

```diff
@@ -171,3 +173,12 @@ fixturator!(
     AnyLinkableHashB64;
     constructor fn new(AnyLinkableHash);
 );
+
+fixturator!(
+    ExternalHash;
+    constructor fn from_raw_32(ThirtyTwoHashBytes);
+);
+fixturator!(
+    ExternalHashB64;
+    constructor fn new(ExternalHash);
+);
```

Comment:
Unrelated really, I started using these in a test but that turned out to be invalid - worth having these available or leave it until next time somebody wants them?